code/game/objects/items/weapons/storage/mre.dm

```diff
@@ -171,6 +171,20 @@ MRE Stuff
 /obj/item/storage/box/fancy/mre/menu11/special // antag item
 	meal_desc = "This one doesn't have a menu listing. How odd. It has the initials \"A.B.\" written on the back."
 
+/obj/item/storage/box/fancy/mre/menu12_kois
+	name = "k'ois MRE"
+	meal_desc = "This one is menu 12, k'ois. DO NOT CONSUME IF YOU ARE NOT VAURCA. Contains k'ois. Only safe for consumption by Vaurca."
+	icon_state = "crayonmre"
+	main_meal = /obj/item/storage/box/fancy/mrebag/menu12_kois
+	starts_with = list(
+		/obj/item/storage/box/fancy/mrebag/dessert/menu12_kois = 1,
+		/obj/item/reagent_containers/food/snacks/koisbar_clean = 1,
+		/obj/random/mre/spread = 1,
+		/obj/random/mre/drink = 1,
+		/obj/item/reagent_containers/food/condiment/small/packet/toothpaste = 1,
```

Comment:
not sure about this, toothpaste is considered alcohol to vaurca, not a daily food

Reply:
Perhaps as an alternate contraband MRE with unprocessed K'ois.

There is already vaurca snacks such as phoron candy, so having the raw grinded phoron reagent has precedence